app/models/billable_metric.rb

```diff
@@ -60,7 +60,7 @@ def aggregation_type=(value)
   end
 
   def active_groups
-    scope = groups.order(created_at: :asc)
+    scope = groups
```

Comment:
Note: This change as no impact anymore on the application as groups are not used anywhere in the app.